client/components/mobile/Explorer.jsx

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import Sidebar from './Sidebar';
 import ConnectedFileNode from '../../modules/IDE/components/FileNode';
 
 
-const Explorer = ({ id, canEdit, onPressClose }) => (
-  <Sidebar title="Files" onPressClose={onPressClose}>
-    <ConnectedFileNode id={id} canEdit={canEdit} onClickFile={() => onPressClose()} />
-  </Sidebar>
-);
+const Explorer = ({ id, canEdit, onPressClose }) => {
+  const { t, i18n } = useTranslation();
```

Comment:
I don't think you need the `i18n` prop here, because the `t` prop is the only one you use. What happens if you remove it?

Reply:
Done